R/favorites.R

```diff
@@ -1,22 +1,21 @@
-#' Get tweets data for statuses favorited by one or more target users.
+#' Get tweets favorited by one or more users
 #'
-#' Returns up to 3,000 statuses favorited by each of one or more
-#' specific Twitter users.
+#' Returns up to 3,000 tweets favorited for each `user`.
 #'
 #' @inheritParams TWIT_paginate_max_id
 #' @inheritParams get_timeline
 #' @return A tibble with one row for each tweet.
 #' @examples
 #' \dontrun{
-#'
-#' ## get max number of statuses favorited by KFC
-#' kfc <- get_favorites("KFC", n = 3000)
+#' # get likes for a single user
+#' kfc <- get_favorites("KFC", n = 600)
```

Comment:
Perhaps make the examples faster by reducing to n = 200 to make just one request?

Reply:
I already made it five times faster 😛 But yes, smaller is better. There's probably no reason to ever set `n` in examples, unless we specifically want to show pagination.